OpenRA.Mods.Common/Widgets/Logic/MusicPlayerLogic.cs

```diff
@@ -36,7 +39,10 @@ public MusicPlayerLogic(Widget widget, Ruleset modRules, World world, Action onE
 			BuildMusicTable();
 
 			Func<bool> noMusic = () => !musicPlaylist.IsMusicAvailable || musicPlaylist.CurrentSongIsBackground || currentSong == null;
-			panel.Get("NO_MUSIC_LABEL").IsVisible = () => !musicPlaylist.IsMusicAvailable;
+			panel.Get("NO_MUSIC_INSTALLED_LABEL").IsVisible =
+				!world.Map.Options.OnlyMapMusic && !musicPlaylist.IsMusicAvailable ? True : False;
```

Comment:
The expressions have constant values once the map is loaded. If this is too out of place, I could replace it.